README.adoc

```diff
@@ -107,6 +107,8 @@ manner that can be evaluated in a shell. If a `service` is specified,
 only the information for that service is displayed. When
 `--script-readable` is specified the output is in `key=value` format.
 The supported services are: Docker; OpenShift.
```

Comment:
@budhrg : although this should not be part of the PR, but while reviewing, I noticed following line can be improved

```
The supported services are: Docker; OpenShift.
```
to 
```
The supported services are: Docker, OpenShift.
```